kuksa-client/kuksa_client/grpc/__init__.py

```diff
@@ -858,6 +858,11 @@ def subscribe(self, *, entries: Iterable[SubscribeEntry], **rpc_kwargs) -> Itera
 
     @check_connected
     def authorize(self, *, token: str, **rpc_kwargs) -> str:
+        """
+        Parameters:
+            rpc_kwargs
```

Comment:
Even if we would have a naming convention - as we not yet are consistent - would it be an idea here to mention in a comment that token shall be the actual token, not a file path.

Reply:
Yes, good point